go/kbfs/kbfsgit/runner_test.go

```diff
@@ -30,6 +30,7 @@ type testErrput struct {
 }
 
 func (te testErrput) Write(buf []byte) (int, error) {
+	te.t.Helper()
```

Comment:
Why this change now?

Reply:
I was trying to comb through the logs and couldn't find the actual failing call because this one kept getting in the way.